teams/5/settings.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="settings.css">
+  <title>Settings</title>
+</head>
+
+
+
+<body>
+    <nav class="navbar">
+      <div class="first">
+        <p>nav</p>
+
+        <ul class="top">
+          <p>ul</p>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+        </ul> 
+
+        <ul class="bottom">
+          <p>ul</p>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li><i>i</i><a href="#">ul li a</a></li>
+          <li>    
+            <img src="ms.jpg" alt="img" width="50px" height="50px">
+          </li>
+        </ul> 
+      </div>
+      <div class="second">
+        <ul>
+          <p>ul</p>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span> </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+          <li><div class="row"><i>i</i><a href="#">ul li a</a> <span>span</span>  </div></li>
+   
+        </ul>  
+       
+         <ul>       
+          <p>ul</p>  
+          <li><div class="spaceTopTwo"><i>i</i><a href="#">ul li a</a></div></li>
+          <li><div><i>i</i><a href="#">ul li a</a></div></li>
+         <li>
+           <div class="row"></div>
+             <div  class="spaceTopTwo">
+             <h2>h2</h2>
```

Comment:
what is this h2 and p?

Reply:
I don't understand your question, kindly clarify.

Reply:
from my understanding, in this part you create the bottom part for sub nav, which includes: 
1 name
1 email
1 icon for log out

This should be translated into: 
2 span + 1 i

But from your code, there are: 
2 i => not sure what is this
then 1 h2 => should be span
then 1 p => should be span
then 1 i => this is correct